extension/mergeSort.js

```diff
@@ -0,0 +1,38 @@
+function mergeSort( initialSet = [] ){
+  if( initialSet == null ){
+    return 'Cannot sort null'
```

Comment:
I think you should raise errors here instead of return an error message as a string